docs/whatsnew/4.3.rst

```diff
@@ -38,6 +38,18 @@ made much more precise (down to the milli-arcsec level).  To help this, the
 Furthermore, by ensuring expensive calculations of the precession-nutation
 matrix are only done once, the transformations have been sped up considerably.
 
+.. _whatsnew-4.3-threadsafe:
+
+Improvements in making Astropy threadsafe
+=========================================
+
+To help use of Astropy in software for the Square Kilometer Array, an effort
+has started to make Astropy threadsafe for operation that do not explicitly
```

Comment:
```suggestion
has been started to make parts of ``astropy`` thread-safe for operations that do not explicitly
```

Reply:
Mostly used - Used "has been started" but replaced "parts of"  with "the basic infrastructure of"